Clickstream/Core/Interface/ClickStream.swift

```diff
@@ -145,3 +142,23 @@ extension Clickstream {
         return sharedInstance
     }
 }
+
+#if TRACKER_ENABLED
+extension Clickstream {
+    
+    /// initialise tracker
+    /// - Parameters:
+    ///   - configs: ClickstreamHealthConfigurations
+    ///   - commonProperties: CSCommonProperties
+    ///   - dataSource: TrackerDataSource
+    ///   - delegate: TrackerDelegate
+    public func setTracker(configs: ClickstreamHealthConfigurations, commonProperties: CSCommonProperties,
+                           dataSource: TrackerDataSource, delegate: TrackerDelegate) {
+        Tracker.initialise(commonProperties: commonProperties, healthTrackingConfigs: configs,dataSource: dataSource, delegate: delegate)
+    }
+    
+    public func getTracker() -> Tracker? {
+        return Tracker.sharedInstance
+    }
```

Comment:
Since the only way to access/setup `Tracker` is via `Clickstream` class, hence this function will be used to update the CommonProperties of Tracker

`self.clickstream?.getTracker()?.updateCommonProperties(commonProperties: commonProperties)`